src/codecoverage/backend/codecoverage_backend/api.py

```diff
@@ -5,10 +5,10 @@
 
```

Comment:
We could actually remove the Phabricator endpoint in codecoverage/backend, https://github.com/mozilla/release-services/issues/1992.

Reply:
I did a PR for that, and rebased this one on top (first 2 commits): no more `src/codecoverage/backend` changes here.